java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -152,130 +130,380 @@ private void execute(String signature, SquidClassLoader classLoader) {
   }
 
   @VisibleForTesting
-  void executeInstruction(BytecodeCFGBuilder.Instruction instruction) {
+  void executeInstruction(Instruction instruction) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3776)